src/Stripe/Infrastructure/Requestor.cs

```diff
@@ -150,18 +148,15 @@ private static string ExecuteRequest(HttpRequestMessage requestMessage)
             throw BuildStripeException(response.StatusCode, requestMessage.RequestUri.AbsoluteUri, responseText);
         }
 
-#if !PORTABLE
         private static async Task<string> ExecuteRequestAsync(HttpRequestMessage requestMessage)
         {
             var response = await HttpClient.SendAsync(requestMessage);
 
             if (response.IsSuccessStatusCode)
                 return await response.Content.ReadAsStringAsync();
 
-            // this is not async
-            throw BuildStripeException(response.StatusCode, requestMessage.RequestUri.AbsoluteUri, response.Content.ReadAsStringAsync().Result);
+            throw BuildStripeException(response.StatusCode, requestMessage.RequestUri.AbsoluteUri, await response.Content.ReadAsStringAsync());
```

Comment:
How very kind of you to review this :)... I am fuzzy on the code here, because I am using .Result in my code. I don't think it matters, however, because it's an exceptional case.